frontend/src/components/dashboard/aliases/AliasDeletionButtonPermanent.tsx

```diff
@@ -68,7 +52,7 @@ export const AliasDeletionButtonPermanent = (props: Props) => {
         </p>
         <WarningBanner />
         <hr />
-        <form onSubmit={onConfirm} className={styles.confirm}>
+        <form className={styles.confirm}>
```

Comment:
question (non-blocking): Should this still be a form element at all? Since we're not using the submission flow at all?

Reply:
Changed to a div - thanks!